src/Valleysoft.DockerCredsProvider/CredsProvider.cs

```diff
@@ -116,4 +117,21 @@ private static async Task<ICredStore> GetCredStoreAsync(string registry, IFileSy
 
         throw new CredsNotFoundException($"No matching auth specified for registry '{registry}' in Docker config.");
     }
+
+    private static string ConvertToHostname(string input)
+    {
+        string stripped = input;
+        if(input.StartsWith("http://")) 
+        {
+            stripped = input.Replace("http://","");
```

Comment:
I prefer the use of `string.Empty` over `""` to indicate that the intent is to truly have an empty string and not just a mistakenly defined string that is missing a value.

Reply:
Also, please use a space after commas for method calls.

Reply:
done.